types/webpack-sources/lib/CompatSource.d.ts

```diff
@@ -3,7 +3,7 @@ import Source = require("./Source");
 
 declare class CompatSource extends Source {
     constructor(sourceLike: SourceLike);
-    static from(sourceLike: SourceLike): Source | CompatSource;
+    static from(sourceLike: SourceLike): Source;
```

Comment:
Documentation says the previous type is correct: https://github.com/webpack/webpack-sources?tab=readme-ov-file#public-methods-4

Reply:
`CompatSource` is inherits from `Source` so just return `Source` is ok.

Reply:
@gabritto Another reason is that `webpack-sources` are designed for use in webpack, and the types in webpack are defined as such.